css_tasks/task9/index.html

```diff
@@ -0,0 +1,378 @@
+              <img
+               src="assets/avatar-2-u-231@3x.png"
+              alt="Person"
+              class="customers-element__image"
+              />
+            </picture>
+            <header>
+              <h5 class="customers-element__name">John S.</h4>
+              <h6 class="customers-element__job">Freelancer</h6>
+            </header>
+          </div>
+        </div>
+
+        <div class="customers__element">
+          <p class="customers-element__text section__text">Lorem ipsum dolor sit amet,</br> consectetur adipisicing elit. Doloribus accusamus expedita repellat similique odio aspernatur ex, architecto eaque quo suscipit.</p>
+        <div class="customers-element__wrapper">
+            <picture>
+              <source media="(max-width:1024px)"srcset="assets/avatar-2-u-231@2x.png"/>
+              <img
+               src="assets/avatar-2-u-231@3x.png"
+              alt="Person"
+              class="customers-element__image"
+              />
+            </picture>
+            <header>
+              <h5 class="customers-element__name">Susan W.</h4>
+              <h6 class="customers-element__job">Photographer</h6>
+            </header>
+          </div>
+        </div>
+
+      </div>
+   </section>
+   <section class="picture">
+      <picture>
+        <source media="(max-width:678px)" srcset="assets/u-25-state-0-min.png" />
+        <source
+          media="(min-width:679px) and (max-width:1024px)"
+          srcset="assets/u-25-state-0@2x-min.png"
+        />
+        <img
+          src="assets/u-25-state-0@3x-min.png"
+          alt="Buildings"
+          class="jumbotron__image"
+        />
+      </picture>
+      <div class="container">
+      <header class="section__header picture__header">
+        <h2 class="section__headline picture__headline">stylish axure design</h2>
+        <div class="section__separator-center picture__separator"></div>
+        <p class="section__text picture__text">Use the sections you need, remove the ones you don't.  Create gorgeous prototypes faster than ever!  </p>
+      </header>
+      <button class="picture__button">Download</button>
+      </div>
+   </section>
+   <section class="form container">
+      <header class="section__header">
+        <h2 class="section__headline">contact us</h2>
+        <div class="section__separator-center"></div>
+        <p class="section__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+      </header>
+      <form class="form__container">
+        <div class="form-left-column">
+          <input class= "form-input" type="text" name="name" placeholder="Name" />
+          <input class= "form-input" type="email" name="email" placeholder="Email"/>
+          <input class= "form-input" type="password" name="password" placeholder="Password"/>
+        </div>
+        <div class="form-right-column">
+          <textarea class="form-input" name="message" cols="30" rows="13" placeholder=" Message"></textarea>
```

Comment:
user shouldn`t be able to resize textarea in this case

Reply:
ok